s2i/bin/run

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+source /files/start.sh
```

Comment:
so no way to replace `start.sh`?

Reply:
I guess we could move contents of /files/start.sh to this file. Would that be more suitable?

Reply:
I'll be honest: I don't know. I would probably leave it as it is and wait for some feedback whether it's useful like this or someone wants to do some absolutely crazy stuff.

Reply:
Now that you've pointed it out, I think we can safely move & delete this file. If someone would like to make changes to the startup routine, they can use pre-init script to edit/replace /usr/local/s2i/run. 

AFAIK the reason this file is separated is that it is a leftover from when this had bunch of startup scripts, that were specific to postfix variant.